### PR TITLE
Add zope2-py2 as test environment

### PR DESCRIPTION
**Pull request by viktordick**

### CODE REVIEW

**Comment by viktordick** [comment]
Found the reason why the test was not working with Zope 2:
At some refactoring step I dropped wrapping the user into the `UserFolder`, which breaks in Zope2, see [here](https://github.com/zopefoundation/AccessControl/issues/91).
Not sure if this also had other consequences, like created objects not having correct ownership. I hope not.

File: perfact/zodbsync/zodbsync.py [hunk 1]
```diff
@@ -278,7 +278,7 @@ def create_manager_user(self):
         Make sure the manager user exists.
         """
         userfolder = self.app.acl_users
-        user = userfolder.getUserById(self.manager_user)
+        user = userfolder.getUser(self.manager_user)
```
Comment (jan-jockusch) [inline perfact/zodbsync/zodbsync.py:281]: Why is it that there's no wrapping happening here? All other `.getUser()` calls have the wrapper.

Comment (jan-jockusch) [reply perfact/zodbsync/zodbsync.py:281]: Ah, maybe because `user` is only used for a test.

**Review by jan-jockusch — COMMENTED** [review]
Minor question remaining, otherwise LGTM

**Comment by viktordick** [comment]
Just tried to run the tests with Ubuntu and everything failed. `git init` has no `-b` option yet in 2.25.1. Will revert this change for now.

File: perfact/zodbsync/zodbsync.py (hunk 1 above)
Comment (viktordick) [reply perfact/zodbsync/zodbsync.py:281]: Exactly. `None` has no attribute `__of__`, but we only want to check for `None` here.
BTW, `getUser` and `getUserById` seem to do the same.

Comment (jan-jockusch) [reply perfact/zodbsync/zodbsync.py:281]: Yes, it's quite clear. I should have read the following line a bit more attentively.

**Review by jan-jockusch — APPROVED** [review]
LGTM

**Review by gandie — APPROVED** [review]
Tests work in all envs, code seems fine

lgtm

**Pull request merged**